resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -300,6 +300,17 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Allocation batch delay must be a positive time value.")
       .createWithDefaultString("1s")
 
+  val KUBERNETES_ALLOCATION_DRIVER_READINESS_TIMEOUT =
+    ConfigBuilder("spark.kubernetes.allocation.driver.readinessTimeout")
+      .doc("Time to wait for driver pod to get ready before creating executor pods. This wait " +
+        "only happens on application start. If timeout happens, executor pods will still be " +
+        "created.")
+      .version("3.2.0")
```

Comment:
@cchriswu . Sorry, but could you update this to `3.1.3`? Since K8s is GA at Apache Spark 3.1 and this bug is reported at 2.4.x, I believe we need to backport this to branch-3.1 and release `3.1.3` and `3.2.0` together.

Reply:
Sure . No worries. Updated :)